src/ObjectUtils/include/ObjectUtils/ObjectFile.h

```diff
@@ -67,6 +67,14 @@ class ObjectFile : public SymbolsFile {
 
   [[nodiscard]] virtual bool IsElf() const = 0;
   [[nodiscard]] virtual bool IsCoff() const = 0;
+
+  static constexpr uint64_t kUnknownSymbolSize = std::numeric_limits<uint64_t>::max();
+
+  [[nodiscard]] static bool SymbolInfoLessByAddress(const orbit_grpc_protos::SymbolInfo& lhs,
+                                                    const orbit_grpc_protos::SymbolInfo& rhs);
+
+  static void DeduceDebugSymbolMissingSizes(
```

Comment:
This name is too generic now that the method is for public use. How are the sizes deduced? There can be multiple ways depending on the specific file type, and using the distance from the next symbol can be very wrong if not all symbols are there. Suggestion: `DeduceDebugSymbolMissingSizesAsDistanceFromNextSymbol`.

Reply:
I like it. Done.